frontend/src/component/playground/Playground/PlaygroundForm/PlaygroundCodeFieldset/PlaygroundCodeFieldset.tsx

```diff
@@ -103,6 +107,27 @@ export const PlaygroundCodeFieldset: VFC<IPlaygroundCodeFieldsetProps> = ({
         }
     };
 
+    const handleAutocompleteOnChange = (e: FormEvent, newValue: any) => {
```

Comment:
is newValue any typed? can we be more specific here e.g. string | string[]? as a rule of thumb I'd avoid all any types in newly written code.